### PR TITLE
Create AuthoritativeRegionView with example page

### PR DESCRIPTION
**Pull request by ehunter-usgs**
fixes usgs/hazdev-geoserve-ws#111

### CODE REVIEW

File: src/htdocs/js/geoserve/LocationApp.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ var LocationApp = function (options) {
     options = Util.extend({}, _DEFAULTS, options);
     _url = options.url;
 
-    el = _this.el;
+    el = _this.el || document.createElement('div');
```
Comment (jmfee-usgs) [inline src/htdocs/js/geoserve/LocationApp.js]: View creates an element, so this shouldn't be needed.

**Comment by emartinez-usgs** [comment]
etc/data.json
- Suggestion: Maybe call this file "regiondata.json" since we may want "placesdata.json" as well?

example/AuthoritativeRegionView.html
- Provide output for example both with and without response data.
- Add spaces in H1 in page markup to match page title.
- Add viewport tag.

example/AuthoritativeRegionView.js
- Provide output for example both with and without response data.
- Create view up front with model and leverage model binding to render Xhr results.
- Tabs to spaces

gruntconfig/browserify.js
- Package classes (those in the "geoserve" project) should be listed _before_ dependencies.
- Add ".js" to EXPORTS mapping for package class.

src/htdocs/js/geoserve/AuthoritativeRegionView.js
- Tabs to spaces
- `render` method starts blindly using `regions` object, but that does not appear to ever be defined.
- auth is already defined to be the properties for the first auth feature in the collection. Should not re-reference properties subsequently when rendering.

test/spec/geoserve/AuthoritativeRegionViewTest.js
- When checking `.to.not.throw('Error')`, I think you don't quote the "Error" and rather use a reference like so: `.to.not.throw(Error)`.
- The 'View' describe looks like it is testing the `render` method and should be labeled as such.
- Probably want to test the case when data is returned since tests currently pass but I suspect the render method is not quite right when data is available to render.

**Comment by jmfee-usgs** [comment]
please rebase

**Comment by emartinez-usgs** [comment]
Rebased and merged externally.

**Pull request closed without merging**